spec/features/generic_announcements_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.feature 'Generic Announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:user) { create(:user) }
+
+    context 'As a User' do
+      before do
+        instance.announcements.clear
+        SystemAnnouncement.destroy_all
+        visit announcements_path
+      end
+
+      scenario 'I should not see any announcements if there are none' do
+        it { is_expected.not_to have_selector('div.instance-announcement') }
+        it { is_expected.not_to have_selector('div.system-announcement') }
+      end
+    end
+
+    context 'As a User' do
```

Comment:
Or, if the same setup needs to be done, then let the context share the `before` block.